### PR TITLE
Release notes 2 1 17

### PR DESCRIPTION
**Pull request by jfrontain**
## This PR is a:

- [x ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

When this pull request is merged, it will add release notes for the Magento Open Source 2.1.17 and Magento Commerce 2.1.17 releases.

whatsnew
The [Magento Open Source 2.1.17 Release Notes](https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.17CE.html)  and  [Magento Commerce 2.1.17  Release Notes](https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.17EE.html) provide detailed information about the Magento Open Source 2.1.17 and Magento Commerce 2.1.17 releases.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3980) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3980) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+group: release-notes
+title: Magento Open Source 2.1.17 Release Notes
+---
+
+*	TOC
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]: What is this?

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{:toc}
+
+
+*Release notes published March 26, 2019.*
+
+
+We are pleased to present Magento Open Source 2.1.17. This release includes multiple enhancements to product security.
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]:
```suggestion
We are pleased to present {{site.data.var.ce}} 2.1.17. This release includes multiple enhancements to product security. 
```

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+This release include security enhancements that help close cross-site scripting, arbitrary code execution, and sensitive data disclosure vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. We recommend that you upgrade your Magento software to the latest version as soon as possible.
+
+
+## Highlights
+
+See [Magento Security Center](https://magento.com/security/patches/magento-2.3.1-2.2.8-and-2.1.17-security-update) for a comprehensive discussion of these issues. All exploitable security issues fixed in this release (2.1.17) have been ported to 2.3.1, 2.2.8, 1.14.4.1, and 1.9.4.1, as appropriate. 
+
+### Note about changes to the Authorize.Net extension
+
+Magento’s implementation of the Authorize.Net Direct Post payment method currently uses MD5 based hash for all M1 and M2 installations. As of June 28, 2019, Authorize.Net will stop supporting MD5 based hash usage (announcement).
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]:
```suggestion
Magento’s implementation of the Authorize.Net Direct Post payment method currently uses an MD5-based hash for all M1 and M2 installations. As of June 28, 2019, Authorize.Net will stop supporting MD5-based hash usage (announcement).
```

Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]: Shouldn't "announcement" be linked?

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+group: release-notes
+title: Magento Commerce 2.1.17 Release Notes
+---
+
+*	TOC
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]: What is this?

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+{:toc}
+
+*Release notes published March 26, 2019.*
+
+
+We are pleased to present Magento Commerce  2.1.17. This release includes  multiple enhancements to product security.
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]:
```suggestion
We are pleased to present {{site.data.var.ee}} 2.1.17. This release includes  multiple enhancements to product security. 
```

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+Although this release includes these security enhancements, no confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. We recommend that you upgrade your Magento software to the latest version as soon as possible.
+
+
+## Highlights
+
+See [Magento Security Center](https://magento.com/security/patches/magento-2.3.1-2.2.8-and-2.1.17-security-update) for a comprehensive discussion of these issues. All exploitable security issues fixed in this release (2.1.17) have been ported to 2.3.1, 2.2.8, 1.14.4.1, and 1.9.4.1, as appropriate. 
+
+### Note about changes to the Authorize.Net extension
+
+Magento’s implementation of the Authorize.Net Direct Post payment method currently uses MD5 based hash for all M1 and M2 installations. As of June 28, 2019, Authorize.Net will stop supporting MD5 based hash usage (announcement).
```
Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]:
```suggestion
Magento’s implementation of the Authorize.Net Direct Post payment method currently uses an MD5-based hash for all M1 and M2 installations. As of June 28, 2019, Authorize.Net will stop supporting MD5-based hash usage (announcement).
```

Comment (jeff-matthews) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]: Shouldn't "announcement" be linked?

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md (hunk 1 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]: removed. legacy ick.

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md (hunk 3 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]: changed

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md (hunk 4 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]: legacy ick

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md (hunk 5 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]: changed

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md (hunk 6 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]: removed. redundant

**Pull request closed without merging**